domain/timeline/src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/timeline/service/TimelineServiceTest.kt

```diff
@@ -2,27 +2,38 @@ package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.ser
 
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.catchThrowableOfType
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.ExtendWith
-import org.mockito.InjectMocks
 import org.mockito.Mock
 import org.mockito.junit.jupiter.MockitoExtension
 import org.mockito.kotlin.any
 import org.mockito.kotlin.given
 import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoInteractions
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.TimelineEvent
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.TimelineNotFoundException
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.aValidTimeline
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.aValidTimelineEvent
 
 @ExtendWith(MockitoExtension::class)
 class TimelineServiceTest {
 
-  @InjectMocks
   private lateinit var service: TimelineService
 
   @Mock
   private lateinit var persistenceAdapter: TimelinePersistenceAdapter
 
+  @Mock
+  private lateinit var prisonTimelineService: PrisonTimelineService
+
+  private val callPrisonApiEnabled = true
+
+  @BeforeEach
+  fun setup() {
+    service = TimelineService(persistenceAdapter, prisonTimelineService, callPrisonApiEnabled)
```

Comment:
Once we remove the `callPrisonApiEnabled` feature toggle, I'll revert this test back to use `@InjectMocks`.